requirements.txt

```diff
@@ -20,10 +20,15 @@ pandoc==2.0a4
 psycopg2==2.8.3
 pypandoc==1.6.4
 python-dateutil==2.8.1
-pytz==2021.1
+pytz~=2019.2
 textstat==0.6.2
 whitenoise==4.1.3
 wordfreq==2.3.2
 -e git+https://github.com/IMSGlobal/caliper-python@1.1.0.2#egg=imsglobal_caliper
 -e git+https://github.com/ouhouhsami/django-progressbarupload#egg=django-progressbarupload
 -e git+https://github.com/cast-org/django-session-timeout#egg=django-session-timeout
+
+requests~=2.25.1
+aniso8601~=9.0.1
+rfc3986~=1.4.0
+setuptools~=40.8.0
```

Comment:
I don't understand how any of the changes in requirements.txt are related to this PR